### DISCUSSION TITLE
Better support for storing date and time

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#10130] [Feature Requests & Ideas]
Currently, it's not possible to store only the date or only the time in the date/time fields. It would be great if we can import support in this aspect.

**Comment by JarrodMFlesch** [comment]
Do you mean the display of the field? I feel like this is already possible with admin.dateAndTimeOptions on the date field. In the db you'll want to still store a date which will always have both.

**Reply by tyteen4a03** [reply]
> Do you mean the display of the field? I feel like this is already possible with admin.dateAndTimeOptions on the date field. In the db you'll want to still store a date which will always have both.

Not just the display, but also the storage and filtering (e.g. filtering by `equals` on a `dateOnly` field will not do what you expect)
